source/voice_control/voice_remote_cloud_assistant.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: "Creating a cloud Assist pipeline"
+---
+
+In Home Assistant, the Assist pipelines are made up of various components that together form a voice assistant.
+
+You can use Assist out of the box by typing into it's text field. But the main goal is to use a voice command. A speech-to-text pipeline is used so that you can speak to the assistant. For Assist to be able to talk to you, you need to set up a text-to-speech engine. You can use these engines fully locally on your own hardware. To learn how, [follow this procedure](/voice_control/create_wake_word/).
```

Comment:
```suggestion
You can use Assist out of the box by typing into its text field. But the main goal is to use a voice command. A speech-to-text engine is used so that you can speak to the assistant. For Assist to be able to talk to you, you need to set up a text-to-speech engine. You can use these engines fully locally on your own hardware. To learn how, [follow this procedure](/voice_control/create_wake_word/).
```